wcomponents-theme/unbuilt.package.json

```diff
@@ -18,6 +18,8 @@
 		"handlebars": "~4.0.5",
 		"i18next": "~3.4.1",
 		"i18next-xhr-backend": "~1.0.1",
```

Comment:
What if we put a comment in the theme-parent pom.xml above the intern version parameter.